PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl/TouchJump.rbxmx

```diff
@@ -55,8 +56,8 @@ function TouchJump:Create(parentFrame)
 	JumpButton.ImageRectOffset = Vector2.new(176, 222)
 	JumpButton.ImageRectSize = Vector2.new(174, 174)
 	JumpButton.Size = UDim2.new(0, jumpButtonSize, 0, jumpButtonSize)
-	JumpButton.Position = isSmallScreen and UDim2.new(1, jumpButtonSize * -2.25, 1, -jumpButtonSize - 20) or
-		UDim2.new(1, jumpButtonSize * -2.75, 1, -jumpButtonSize - 120)
+	JumpButton.Position = isSmallScreen and UDim2.new(1, -origJumpButtonSize - 10, 1, -origJumpButtonSize) or
```

Comment:
I've tested it on my phone, an iPhone 4s with a tiny screen, and a Samsung tablet with a normal aspect ratio. I can try a few more to make sure it looks ok on more devices.